src/browser/components/MattermostView.jsx

```diff
@@ -24,7 +24,7 @@ const preloadJS = `file://${remote.app.getAppPath()}/browser/webview/mattermost_
 const ERR_NOT_IMPLEMENTED = -11;
 const U2F_EXTENSION_URL = 'chrome-extension://kmendfapggjehodndflmmgagdbamhnfd/u2f-comms.html';
 
-const appIconURL = `file:///${remote.app.getAppPath()}/assets/appicon.png`;
```

Comment:
should we downsize all OSs?

Reply:
It seems to only be an issue on Windows, and I think that's the only OS using an image we provide. I'm pretty sure Mac just uses the icon from the application itself.